internal/controller/postgrescluster/postgres.go

```diff
@@ -259,6 +259,13 @@ func (r *Reconciler) reconcilePostgresUsers(
 	if err == nil {
 		err = r.reconcilePostgresUsersInPostgreSQL(ctx, cluster, instances, users, secrets)
 	}
+	if err == nil {
+		// TODO(cbandy): If I were looking for pgAdmin stuff, I wouldn't expect
```

Comment:
Is there a better place to put this code? I think you lay out the case for why it's here, so I'm not sure there's a TODO here.

Reply:
Agreed. I dropped the TODO.